public/documentation/ios.md

```diff
@@ -0,0 +1,176 @@
+
+    * Create `XWalkExtensions` section in the `Info.plist` with `Dictionary` type, then add an entry with `xwalk.example.echo` as key and `EchoExtension` as value.
+      ![Info.plist](https://cloud.githubusercontent.com/assets/700736/7226047/58728d94-e777-11e4-9fd4-8d23a24d981f.png)
+
+    This indicates that the extension `EchoExtension` will be exported with the object name: `xwalk.example.echo` in JavaScript.
+
+4. Bundle The Extension With The Application
+
+  * Modify the `manifest.plist` to add the extension configuration to the `Echo` application.
+
+    * Add `xwalk_extensions` section with Array type which describes the extensions that should be loaded within the `Echo` application;
+
+    * And add `xwalk.example.echo` with String type as an entry of `xwalk_extensions`, which indicates that the extension `xwalk.example.echo` should be loaded into the JavaScript context of `Echo` project.
+      ![manifest2](https://cloud.githubusercontent.com/assets/700736/7226213/3ef59a9e-e779-11e4-822f-1ef6775723ad.png)
+
+  * Then you can build and run the application to test. If everything goes well, you can see an extra line: `Echo from native: Hello World!` displayed on the screen.
+
+For further information please read the [Getting Started Guide](https://github.com/crosswalk-project/crosswalk-ios/wiki/Getting-Started-With-Crosswalk-for-iOS), and other articles on the [Wiki](https://github.com/crosswalk-project/crosswalk-ios/wiki).
+
+## Community
+* Follow the [crosswalk-help](https://lists.crosswalk-project.org/mailman/listinfo/crosswalk-help) mailing list to ask questions
+* Follow the [crosswalk-dev](https://lists.crosswalk-project.org/mailman/listinfo/crosswalk-dev) mailing list for development updates
+* Find us on IRC: #crosswalk on freenode.net
+
+## Demos
+There're 3 built-in demos in the project:
+
+* [Sample](Demos/Sample)
+
+	A simple demo which shows the basic ways of XWalkView embedding, Crosswalk Extension implementation, configuration of extension and application, etc.
+
+* [CordovaPluginDeviceDemo](Demos/Cordova/CordovaPluginDeviceDemo)
+
+	A demo to show the way to integrate Cordova Plugin with the Crosswalk Cordova Extension support, and the usage of `apache.cordova.device` plugin.
+
+* [CordovaPluginFileDemo](Demos/Cordova/CordovaPluginFileDemo)
+
+	Another Cordova Plugin demo, which is imported from https://github.com/Icenium/sample-file.git, which demostrates the usage of `apache.cordova.file` plugin.
+
+NOTICE: Try them after the project's submodules get initialized, using:
+
+  ```bash
+  git submodule update --init --recursive
+  ```
+
+You may also try out the [HexGL-iOS Demo](https://github.com/jondong/HexGL-iOS) written in HTML5 with the Crosswalk Extension support to evaluate the performance and extensibility of the Crosswalk framework.
+
+## Licence
```

Comment:
link over to:  https://github.com/crosswalk-project/crosswalk/blob/master/LICENSE